frontend/src/core/user/components/UserMenu.test.js

```diff
@@ -8,38 +8,43 @@ import { UserMenuBase } from './UserMenu';
 import { findLocalizedById } from 'test/utils';
 
 
+function createShallowUserMenu({
+    isAdmin = false,
+    isReadOnly = false,
+    isTranslator = true,
+    isAuthenticated = true,
+    locale = 'mylocale',
+    project = 'myproject',
+    resource = 'myresource',
+} = {}) {
+    return shallow(
+        <UserMenuBase
+            isReadOnly={ isReadOnly }
+            isTranslator={ isTranslator }
+            user={ { isAuthenticated: isAuthenticated, isAdmin: isAdmin } }
+            parameters={ { locale: locale, project: project, resource: resource } }
```

Comment:
There's a nice trick with ES6: if the name of the key and the name of the variable are the same, you can shorten the syntax with just the key. For example, in this case, you could write: `{ locale, project, resource }`.

Reply:
Oh, that's cool! Thanks for the pro-tip!